client/src/use/useStyling.ts

```diff
@@ -35,82 +55,165 @@ export default function useStyling({ markChangesPending }: UseStylingParams) {
     strokeWidth: 3,
     opacity: 1.0,
     color: 'type',
+    fill: false,
   };
   const selected: Style = {
     ...standard,
     color: vuetify.preset.theme.themes.dark.accent as string,
     strokeWidth: 5,
     opacity: 1.0,
+    fill: false,
   };
   const disabled: Style = {
     ...standard,
     color: 'type',
     strokeWidth: 0.5,
     opacity: 0.45,
+    fill: false,
   };
-  // Colors provided for the different Types
   const stateStyling: StateStyles = { standard, selected, disabled };
-  const typeColors = [
-    colors.red.accent1,
-    colors.yellow.darken3,
-    colors.purple.lighten3,
-    colors.green.lighten3,
-    colors.yellow.lighten3,
-    colors.purple.darken3,
-    colors.green.darken3,
-  ];
-
-  function loadTypeColors(list?: Record<string, string>) {
-    if (list) {
+
+  /**
+   * Generates a color pallette to be used for mapping type names to colors.
+   * It generates colors from a rainbow spectrum and then takes a dark and lighter version
+   * of each color.
+   * @param {int} numColors - number of colors to attempt to generate the higher
+   * the number the more similar the colors will be.  Cyan like colors will be filtered out,
+   * so numColors isn't a guarantee of x*3 (normal, dark, light) colors.
+   */
+  function generateColors(numColors: number) {
```

Comment:
This is a utility function and doesn't need to have access to any of the state in the parent scope.  I agree that keeping this function in this file makes perfect sense, but I think you can hoist it up above the composition function to avoid scope pollution.

In the general case, function declarations within function declarations should be avoided.